.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
```

Comment:
I personally don't have a problem with adding this per se. It's fairly standard to add this to `.editorconfig`, as you can see from a sampling of major oss projects [here](https://github.com/editorconfig/editorconfig/wiki/Projects-Using-EditorConfig).